create-convex/src/index.ts

```diff
@@ -245,14 +246,14 @@ async function init() {
   await writeCursorRules(root, { verbose });
 
   const pkg = JSON.parse(
-    fs.readFileSync(path.join(root, `package.json`), "utf-8"),
+    fs.readFileSync(path.join(root, `package.json`), "utf-8")
```

Comment:
These changes seem to conflict with what Prettier does (Prettier is installed in this repository). Could you please revert them?

Reply:
Thanks!